lib/manager/flux/readme.md

```diff
@@ -1,6 +1,9 @@
-Checks YAML manifests for [Flux](https://fluxcd.io/) `HelmRelease` resources and extracts dependencies for the `helm` datasource.
+This manager parses [Flux](https://fluxcd.io/) YAML manifests and:
 
-The `flux` manager will only extract `HelmRelease` resources linked to `HelmRepository` sources.
+1. Extracts `helm` dependencies from `HelmRelease` resources
+2. Extracts `github-releases` dependencies from system manifests (`flux-system/gotk-components.yaml` files) and regenerates them when new versions of Flux are available
+
```

Comment:
```suggestion

Updating system manifests requires that the `flux` tool be pre-installed or you are running a Docker image based off [containerbase/buildpack](https://github.com/containerbase/buildpack), such as the official Renovate images, and have `binarySource=install` configured.

```

Reply:
Added this a bit lower in the docs (to match the order of the features described at the top of the file) and separated the two options into bullet points to avoid any doubt about whether `binarySource=install` is required for both options or just the second option.